docs/reference/contributing/guidelines/design_guidelines.md

```diff
@@ -141,10 +202,10 @@ A general module can be split into two APIs, the frontend (or user API) and the
      */
      osStatus lock(uint32_t millisec=osWaitForever);
     ```
-
+    
 - The doxygen of each class's header file must contain a use example using `@code` and `@endcode`.
```

Comment:
A bit out of scope of this specific PR, but related. This needs to be discussed more with whoever will be maintaining Mbed OS examples going forward. I'm all for having examples in the Doxygen, but as we currently have our code maintained the Doxygen code and code snippets found in related examples tend to drift out of sync.